gradle.properties

```diff
@@ -3,6 +3,8 @@ org.gradle.welcome=never
 # Optional - set custom build version
 # version=24.5.6-acme
 # versionappendcommit=true
+org.gradle.dependency.verification=lenient
```

Comment:
don't leave this in by accident (though throwing a comment in front would be nice since I am constantly adding this)